src/runtime/components/Select.vue

```diff
@@ -113,8 +117,22 @@ const ui = computed(() => tv({ extend: select, slots: props.ui })({
 
 const groups = computed(() => props.items?.length ? (Array.isArray(props.items[0]) ? props.items : [props.items]) as SelectItem[][] : [])
 
+function onUpdate(value: any) {
+  const event = new Event('change', { target: { value } })
+  emits('change', event)
+
+  emitFormChange()
+  emitFormInput()
```

Comment:
Will validate the form twice. I'll look into implementing a `dirty` check on the form component to ignore validation requests when the value has not changed to improve performances.